src/PostgREST/DbStructure/Relation.hs

```diff
@@ -32,45 +29,36 @@ data Relation = Relation
   , relColumns  :: [Column]
   , relFTable   :: Table
   , relFColumns :: [Column]
-  , relType     :: Cardinality
-  , relLink     :: Link -- ^ Constraint on O2M/M2O, Junction for M2M Cardinality
+  , relCard     :: Cardinality
```

Comment:
```haskell
  , relForeignTable   :: Table
  , relForeignColumns :: [Column]
  , relCardinality    :: Cardinality
```

could we rename it like that? Would improve readability a lot, I think. I started reading the code above, before I came by the type definition - and I first wondered, what that "card" of a relation was.

I think it would be great if down the road we could make the naming of our record fields consistently:
- have a good, abbreviated prefix to help with `RecordWildCards`. `Relation` already has a good prefix, but for example we will need to improve on `ApiRequest.iXXX`.
- use a non-abbreviated name itself. `relCard` is kinda ookaaay in itself. But the `F` is really hard to spot.